proto/storage/compliance_operator_v2.proto

```diff
@@ -180,3 +180,22 @@ message ComplianceOperatorProfileClusterEdge {
     string        profile_uid = 3[(gogoproto.moretags) = 'search:"Compliance Profile UID,hidden"'];
     string        cluster_id  = 4[(gogoproto.moretags) = 'search:"Cluster ID,hidden" sql:"fk(Cluster:id),no-fk-constraint,type(uuid)"'];
 }
+
+// ComplianceOperatorScanSettingBindingV2 associates a scan setting with one or more compliance profiles
+// Next Tag: 6
+message ComplianceOperatorScanSettingBindingV2 {
```

Comment:
If we create a single configuration that runs on multiple clusters, each cluster will have an occurrence of the name.  So the name, unfortunately, cannot be unique.

We need the cluster_id as a searchable column for SAC purposes.
We should also pull in labels and annotations for completeness.
```
map<string, string> labels      = 4;
map<string, string> annotations = 5;
```